### ISSUE TITLE
How to randomize data in ngid/canvas

### ISSUE THREAD

**Issue by subhasispanda98** [chigix/linked-data-graphic#27]
How to randomize data in ngid/canvas. While i try to change object its not updating view. please help

**Comment by chigix** [comment]
@aysha-statvalu 
Sorry that replies late. Could you give me a sample snippet showing how you expect to "randomize data in ngld/canvas"?

The rendered data is open in a `GraphContainer` interface:
https://github.com/chigix/linked-data-graphic/blob/d77f9d8851f518a04fcbb17802676710195caf18/projects/linked-data-graphic/src/lib/graph.container.ts#L65

Each Container of Graph Data in rendering can be accessed from the `d3Graph` property in the `ngld/canvas` component object:
https://github.com/chigix/linked-data-graphic/blob/d77f9d8851f518a04fcbb17802676710195caf18/projects/linked-data-graphic/src/lib/canvas.component.ts#L102

Hope this introduction helps.

**Comment by subhasis98** [comment]
that thing i fixed now i want to give plus and minus button for zoom in and zoom out how to do that?

**Comment by chigix** [comment]
You want to attach new buttons to control the canvas behaviour?

**Comment by subhasis98** [comment]
yes i want to control zoom in and zoom out canvas behaviour with buttons

**Comment by chigix** [comment]
This is exactly nice to expose the following event listener as a general method to access:
https://github.com/chigix/linked-data-graphic/blob/d77f9d8851f518a04fcbb17802676710195caf18/projects/pan-zoom-directive/src/lib/pan-zoom.directive.ts#L148

Would you be interested in implementing it as a contribution?
